docs/streams/developer-guide/app-reset-tool.html

```diff
@@ -117,6 +118,9 @@ <h2>Step 1: Run the application reset tool<a class="headerlink" href="#step-1-ru
                                         bootstrap.servers, as the reset tool
                                         would no longer access Zookeeper
                                         directly.
+--force                               Force the removal of members of the consumer group
```

Comment:
nit: `Force the removal of` -> `Force removing`

Reply:
Fixed~